index.html

```diff
@@ -61,13 +84,11 @@
             </ul>
           </nav>
           <nav class="navbar">
-            <ul class="links-1">
-              <li class="link1"><a href="./about.html">About</a></li>
-            </ul>
             <img
               src="./assets/images/hamburg.png"
               alt="nav-section"
               class="hamburg-icon"
+              id="menu-close-btn"
             />
           </nav>
         </section>
```

Comment:
- [ ] Great work on the `Main program` section, however, some cards have less space width than others, kindly make sure all the cards in this section have the same sizes so they look semantic

|Your website|Provide design|
|-|-|
|![Screenshot_20230322_091151](https://user-images.githubusercontent.com/25227022/226854923-64945985-ec5d-45f8-b576-b7359b662887.png)|![Screenshot_20230322_091203](https://user-images.githubusercontent.com/25227022/226854922-89c28f3c-caf6-4df5-be5e-79a678b0c963.png)|